### DISCUSSION TITLE
RaspberryMatic Thermostat Modus

### DISCUSSION THREAD

**Discussion by lukibazuki** [SukramJ/aiohomematic#666] [Q&A] [answered]
In RaspberryMatic sind die einzelnen Modis Auto oder Manual. Im HomeAssistant wird der Modus "Manual" jedoch als "Heizen" angezeigt.
Gibt es eine Möglichkeit dies anzupassen?

**Comment by SukramJ** [comment, marked as answer]
Nein, das ist HA Standard.
